pages/new-collective-page.js

```diff
@@ -166,9 +176,21 @@ class NewCollectivePage extends React.Component {
                   isAdmin={Boolean(LoggedInUser && LoggedInUser.canEditCollective(collective))}
                   isRoot={Boolean(LoggedInUser && LoggedInUser.isRoot())}
                   onPrimaryColorChange={onPrimaryColorChange}
+                  step={step}
+                  mode={mode}
                 />
               )}
             </CollectiveThemeProvider>
+            {LoggedInUser && mode === 'onboarding' && process.env.ONBOARDING_MODAL && (
```

Comment:
Best practice is to parse it in place as boolean, because with the current code, setting `ONBOARDING_MODAL=false` will not do what you think it does. See `parseToBoolean` in utils.

Reply:
You're right, it wasn't doing what I expected! Looks like it's actually working now when I change the flag. Thanks!